tests/format/js/explicit-resource-management/jsfmt.spec.js

```diff
@@ -25,26 +26,19 @@ run_spec(import.meta, ["babel", "flow", "typescript"], {
       "valid-module-block-top-level-await-using-binding.js",
     ],
     typescript: [
-      "for-await-using-of-comments.js",
       "invalid-duplicate-using-bindings.js",
-      "invalid-script-top-level-using-binding.js",
-      "using-declarations.js",
-      // "valid-for-await-using-binding-escaped-of-of.js",
-      // "valid-for-using-binding-escaped-of-of.js",
-      "valid-module-block-top-level-using-binding.js",
-      "valid-using-binding-basic.js",
-      "valid-using-binding-escaped.js",
-      "valid-using-binding-non-bmp.js",
-      "valid-using-binding-using.js",
+      "invalid-using-declaration-in-for.js",
 
-      "valid-await-using-binding-basic.js",
-      "valid-await-using-binding-escaped.js",
-      "valid-await-using-binding-non-bmp.js",
-      "valid-await-using-binding-using.js",
+      // https://github.com/typescript-eslint/typescript-eslint/blob/4f34d0ba34474926ba1eed623704b583a037f886/packages/typescript-estree/src/convert.ts#L1035-L1050
+      "valid-for-using-declaration.js",
       "valid-await-using-comments.js",
-      "valid-for-lhs-await-as-identifier.js",
+      "using-declarations.js",
```

Comment:
typescript-estree throws for these files (tsc doesn't throw).

Reply:
We can ignore edge cases.